GDevelop.js/__tests__/GDJS.js

```diff
@@ -378,10 +377,10 @@ describe('libGD.js - GDJS related tests', function () {
       // was not generated for this object,
       // but generated for the Sprite supporting this capability.
       expect(code).toMatch(
-        'gdjs.SceneCode.GDMySpriteObjects1[i].enableEffect("MyEffect", true);'
+        'gdjs.SceneCode.GDMySpriteObjects1[i].getBehavior(\"Effect\").enableEffect(\"MyEffect\", true);'
       );
-      expect(code).toMatch(
-        '/* Object with unsupported capability - skipped. */'
+      expect(code).toEqual(expect.not.stringContaining(
+        'gdjs.SceneCode.GDMyFakeObjectWithUnsupportedCapabilityObjects1[i].getBehavior(\"Effect\")')
       );
```

Comment:
>> But, maybe groups that contain at least one object that match should be display (as code generation allows it for behaviors) ?
>
>Is this really supported by code generation?

According to this test, objects from a group that don't have the behavior are just ignored. Noting is generated for them not even a comment. Probably because it's considered as a common thing.